DH_Interface/Classes/DHTab_Input.uc

```diff
@@ -7,6 +7,7 @@ class DHTab_Input extends ROTab_Input;
 
 var automated DHmoNumericEdit           nu_MousePollRate;
 var automated moFloatEdit               fl_IronSightFactor;
+var automated moFloatEdit               fl_IronSightBipodFactor;
```

Comment:
Should just be `BipodFactor` since there's no state where you are using the bipod but not looking down ironsights.